content/workshops/cascading-styles-deep.md

```diff
@@ -90,12 +126,19 @@ with at least a basic understanding in HTML and CSS.
 You don’t need to be an expert to keep up,
 but even the experts are likely to learn something new.
 
-## Schedule
+[Schedule a call »](#contact-us-to-book-a-workshop)
```

Comment:
I wonder if we want to edit this label a bit. The link takes you to the "Contact Us to Book a Workshop" form but I know next to the submit button there is a link to schedule a call. But also, "schedule a call" feels more like we are going to discuss a project... maybe something in the realm of of the link url "Contact us to book a workshop" or add in discuss the details of the workshop or something?